src/Vulkan/vk_video.h

```diff
@@ -0,0 +1,7 @@
+#include <cstdint>
+#include <vk_video/vulkan_video_codec_h264std_encode.h>
+#include <vk_video/vulkan_video_codec_h265std_decode.h>
+#include <vk_video/vulkan_video_codec_h264std_decode.h>
+#include <vk_video/vulkan_video_codecs_common.h>
+#include <vk_video/vulkan_video_codec_h264std.h>
+#include <vk_video/vulkan_video_codec_h265std.h>
```

Comment:
Why does this exist (here)?

Reply:
This is the input for generating Silk.NET.Vulkan.Video. It didn't feel appropriate in Silk.NET.Vulkan, Silk.NET.Vulkan/Video, or anywhere else really seeing as it's an input source.